pkgs/markdown/test/crash_test.dart

```diff
@@ -43,22 +43,22 @@ void main() async {
       var lastStatus = DateTime(0);
       void status(String Function() message) {
         if (DateTime.now().difference(lastStatus) >
-            const Duration(seconds: 30)) {
+            const Duration(seconds: 5)) {
           lastStatus = DateTime.now();
           print(message());
         }
       }
 
       final c = http.RetryClient(http.Client());
-      Future<dynamic> getJson(String url) async {
+      Future<Map<String, dynamic>?> getJson(String url) async {
         final u = Uri.tryParse(url);
         if (u == null) {
           return null;
         }
         try {
           final data = await c.read(u);
           try {
-            return jsonDecode(data);
+            return jsonDecode(data) as Map<String, dynamic>;
```

Comment:
It's possible that pub.dev/api returns some other type of JSON, but in practice, they are always maps!

Another approach would be to return `Object?` and force folks to cast. 🤷